src/main/java/com/coveros/selenified/utilities/CombinedPDFReport.java

```diff
@@ -0,0 +1,129 @@
+package com.coveros.selenified.utilities;
+
+import org.apache.pdfbox.io.MemoryUsageSetting;
+import org.apache.pdfbox.multipdf.PDFMergerUtility;
+import org.apache.pdfbox.pdmodel.PDDocument;
+import org.apache.pdfbox.pdmodel.PDPage;
+import org.apache.pdfbox.pdmodel.PDPageContentStream;
+import org.apache.pdfbox.pdmodel.PDPageTree;
+import org.apache.pdfbox.pdmodel.common.PDRectangle;
+import org.apache.pdfbox.pdmodel.font.PDFont;
+import org.apache.pdfbox.pdmodel.font.PDType1Font;
+import org.apache.pdfbox.pdmodel.interactive.action.PDActionGoTo;
+import org.apache.pdfbox.pdmodel.interactive.annotation.PDAnnotation;
+import org.apache.pdfbox.pdmodel.interactive.annotation.PDAnnotationLink;
+import org.apache.pdfbox.pdmodel.interactive.annotation.PDBorderStyleDictionary;
+import org.apache.pdfbox.pdmodel.interactive.documentnavigation.destination.PDPageDestination;
+import org.apache.pdfbox.pdmodel.interactive.documentnavigation.destination.PDPageFitWidthDestination;
+import org.testng.IReporter;
+import org.testng.ISuite;
+import org.testng.ITestNGMethod;
+import org.testng.log4testng.Logger;
+import org.testng.xml.XmlSuite;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CombinedPDFReport implements IReporter {
+
+    static final float INCH = 72;
+    static final String FILE_NAME= "/AllPDFReports.pdf";
+    private static final Logger log = Logger.getLogger(CombinedPDFReport.class);
+
+    @Override
+    public void generateReport(List<XmlSuite> xmlSuites, List<ISuite> suites, String outputDirectory) {
+        if (Property.generatePDF()) {
+            try {
+                String testReportDirectory = suites.get(0).getOutputDirectory();
+                List<ITestNGMethod> iTestNGMethods = suites.get(0).getAllMethods();
```

Comment:
What if there is more than one suite? I'm pretty sure we need to check for each one, it we'll miss tests